the-collab-lab/tcl-15-smart-shopping-list
# Issue 17

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: Style   |

## Acceptance Criteria

* The add an item screen now looks pretty, professional, and welcoming.

## Description

* Our add item page looks a lot better now with improved  styling and rich colors for a better overall user experience

![Screen Recording 2020-11-27 at 12 07 54](https://user-images.githubusercontent.com/63402676/100442607-922d1880-30a8-11eb-8943-5ffddaddf403.gif)



## Related Issue

closes #17

## Testing Steps / QA Criteria

* Pull down this branch with git pull origin `yk-se-issue-17` and `git checkout yk-se-issue-17`
* Run `npm start` to launch the app to see our new Add Item page 🌟

## Screenshots

<img width="1280" alt="Screenshot 2020-11-27 at 12 04 01" src="https://user-images.githubusercontent.com/63402676/100442113-ceac4480-30a7-11eb-9407-7408933e8a24.png">


<img width="442" alt="Screenshot 2020-11-27 at 12 04 23" src="https://user-images.githubusercontent.com/63402676/100442256-fdc2b600-30a7-11eb-9a9e-63316d295149.png">
